checks/evaluation/permissions/GitHubWorkflowPermissionsStepsNoWrite.yml

```diff
@@ -0,0 +1,19 @@
+short: Checks that GitHub workflows do not have steps with dangerous write permissions
+desc: This rule checks that GitHub workflows do not have steps with dangerous write permissions
+motivation: >
+  Even with permissions default set to read, some scopes having write permissions in their steps brings incurs a risk to the project.
+  By giving write permission to the Actions you call in jobs, external Action you call could abuse them - deendencin on the permissions, this could let them
```

Comment:
`deendencin` typo

Reply:
```suggestion
  By giving write permission to the Actions you call in jobs, external Action you call could abuse them - depending on the permissions, this could let them
```

Reply:
done, thanks